psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -916,6 +925,23 @@ void OEProp::compute()
 
 void OEProp::compute_multipoles(int order, bool transition)
 {
+    MultipolePropCalc::MultipoleOutputType mpoles = mpc_.compute_multipoles(order, transition, true, print_ > 4);
+    for (auto it = mpoles->begin(); it != mpoles->end(); ++it) {
+        std::string name;
+        double total_mpole = 0.0;
+        // unpack the multipole, which is: name, nuc, elec, total, ignore nuc and elec:
+        std::tie(name, std::ignore, std::ignore, total_mpole) = *it;
+        /*- Process::environment.globals["DIPOLE X"] -*/
+        /*- Process::environment.globals["DIPOLE Y"] -*/
+        /*- Process::environment.globals["32-POLE XXXXX"] -*/
+        /*- Process::environment.globals["32-POLE XXXXY"] -*/
+        Process::environment.globals[name] = total_mpole;
+    }
+}
+
+MultipolePropCalc::MultipoleOutputType MultipolePropCalc::compute_multipoles(int order, bool transition,
+                                                                             bool print_output, bool verbose) {
```

Comment:
For clarification, `print_output` signals the difference between a silent function and a normal modest-print-level function, while `verbose` signals additional debug level printing?

This is a bikeshedding comment, so fine for now. I just want to understand the intent in case we ever reform our print levels.

Reply:
print_output = True outputs the unchanged output from the OEProp library.

The Tasklist part of the Prop class had two print levels: int print_ == 2 and print_ == 4, so there was the attempt to implement some private print_level system here.
They were only used for debugging purposes printing extra information about
- the multipole integrals
- the ESP at Nuclei integrals

If there was a debuglevel enum, or a general different logger class with .INFO, .DEBUG etc. functions, the verbose function argument could be dropped.